PP-172: Added doc section in Troubleshooting about m1 and rosetta fix (#19)
rsksmart/ri-aggregation: docs/launch.md

[this thread](https://forums.docker.com/t/how-to-remove-none-images-after-building/7050).

After this, re-install the project. This will trigger sizable downloads, so expect this to take a while.

### Compiling issues M1
***Problem***
The error could be `dyld[xxxx]: missing symbol called`.

> Using mac m1, the clang version (c and c++ compiler) its only for that architecture and the code from rust its compiled depending on that version but some dependencies have only support for the intel x86_64.

Reviewer: ```suggestion
With Mac M1, the clang version (c and c++ compiler) compiles for the ARM architecture only but there are some Rust dependencies that have support for the intel x86_64 only.
```
